doc/installationguide.md

```diff
@@ -30,6 +30,20 @@ These are the parameters that can be configured in the global section:
 }
 ```
 
+-   If you want to use NGSI-LD (experimental):
+
+```javascript
+{
+    host: '192.168.56.101',
+    port: '1026',
+    ngsiVersion: 'ld',
+    jsonLdContext: 'http://context.json-ld'
```

Comment:
Shouldn't you be able to pass multiple contexts ?

Reply:
To handle multiple contexts you can create a file which holds an array of contexts:

```jsonld
{
  "@context": [
        "https://fiware.github.io/tutorials.Step-by-Step/tutorials-context.jsonld",
        "https://fiware.github.io/data-models/context.jsonld",
        "https://uri.etsi.org/ngsi-ld/v1/ngsi-ld-core-context.jsonld"
    ]
}
```

The files `tutorials-context.jsonld`, `context.jsonld` and `ngsi-ld-core-context.jsonld` each a separate context.

I'd assume the context won't bee too large - you just need the Device definitions. Even if it is quite big, in reality you'll only need to load the context(s) once and thereafter it will be cached.